packages/next-app/src/components/FilterDropdown.tsx

```diff
@@ -30,10 +30,10 @@ function DropdownRadioItem({
 
   function handleSelectFilter(value: string) {
     setFilterValue(value);
-    changeSelectedFilters(toSlug(filter.name), value);
+    changeSelectedFilters(toCamelCase(filter.name), value);
   }
   function handleClearFilter() {
-    clearSelectedFilter(toSlug(filter.name));
+    clearSelectedFilter(toCamelCase(filter.name));
```

Comment:
couldn't we simply have the filter name be camel case? I still think this pattern is very error prone